src/Caching/Memory/src/CacheEntry.cs

```diff
@@ -49,7 +51,7 @@ internal class CacheEntry : ICacheEntry
             }
 
             Key = key;
-            _notifyCacheEntryDisposed = notifyCacheEntryDisposed;
+            _notifyCacheEntryCommit = notifyCacheEntryDisposed;
```

Comment:
Can you also rename the parameter?
```suggestion
            _notifyCacheEntryCommit = notifyCacheEntryCommit ;
```

Reply:
Whoops, missed that on the find-replace